src/main/fc/settings.yaml

```diff
@@ -1769,3 +1769,26 @@ groups:
         max: 255
         type: uint8_t
 
+  - name: PG_RCDEVICE_CONFIG
+    type: rcdeviceConfig_t
+    headers: ["io/rcdevice.h"]
+    condition: USE_RCDEVICE
+    members:
+      - name: rcdevice_init_attempts
+        field: initDeviceAttempts
```

Comment:
Please, get rid of this and `rcdevice_init_attempt_interval`. Driver should hardcode some sane values that work with the camera. Having these adjustable parameters just gives the users a chance to set them incorrectly (or for they to become corrupted).

Reply:
Yes, got it.